Scripts/Item Scripts/key.gd

```diff
@@ -0,0 +1,11 @@
+class_name Key extends Item
+
+
```

Comment:
Does this class exist purely for filtering?

Reply:
yes, additional functionality was discussed